### PR TITLE
ci: adding config for stack-validate

### PR DESCRIPTION
**Pull request by talset**

### CODE REVIEW

File: .ci/stack-validate-pipeline-variables.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+# Repos
+#
+
+#. git_repository (required): git@github.com:my/repo_to_validate.git
+#+ Git repository URL containing the stack to validate.
+git_repository: git@github.com:cycloid-community-catalog/stack-prometheus.git
+
+#. git_branch (required): master
+#+ Branch of the stack to validate Git repository.
+git_branch: master
```
Comment (sdurrheimer) [inline .ci/stack-validate-pipeline-variables.yml]: Shouldn't it be `((env))` ?

File: .ci/stack-validate-pipeline-variables.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  USERNAME: ((cycloid_bot_account.login))
+  PASSWORD: ((cycloid_bot_account.password))
+  # TEMPLATE to use in the stack
+  PIPELINE_TEMPLATE: prometheus.yml
+  # VARIABLES to use to create the PIPELINE_TEMPLATE
+  PIPELINE_VARIABLES: variables.sample.yml
+  # Where to create the pipeline
+  ORGANIZATION: cycloid-owl
+  # Tmp name to create the pipeline
+  PIPELINE_NAME: ci-stack-prometheus-master-validate
```
Comment (sdurrheimer) [inline .ci/stack-validate-pipeline-variables.yml]: Shouldn't it be `ci-stack-prometheus-((env))-validate` ?

File: .ci/stack-validate-pipeline-variables.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+
+#
+# Default variables that shouldn't be changed except if needed
+#
+
+#. env (required): master
+#+ Name of the project's environment.
+env: master
```
Comment (sdurrheimer) [inline .ci/stack-validate-pipeline-variables.yml]: Shouldn't it be ((env)) ?

File: .ci/stack-validate-pipeline-variables.yml (hunk 1 above)
Comment (talset) [reply .ci/stack-validate-pipeline-variables.yml]: This file is the variables from https://github.com/cycloidio/cycloid-stacks/blob/stacks/stack-validate/pipeline/variables.sample.yml

we don't have  `((env))` variable, only `git_branch: ($ environment $)` already render by the console

**Pull request merged**